src/main/java/com/google/sps/placeGuide/repository/impl/DatastorePlaceGuideRepository.java

```diff
@@ -0,0 +1,121 @@
+package com.google.sps.placeGuide.repository.impl;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.CompositeFilter;
+import com.google.appengine.api.datastore.Query.CompositeFilterOperator;
+import com.google.sps.placeGuide.PlaceGuide;
+import com.google.appengine.api.datastore.GeoPt;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import org.jetbrains.annotations.Nullable;
+import com.google.sps.placeGuide.repository.PlaceGuideRepository;
+
+/** Class for handling place guide repository using datastore. */
+public class DatastorePlaceGuideRepository implements PlaceGuideRepository{
+
+  private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  public static final String ENTITY_KIND = "PlaceGuide";
+  public static final String NAME_PROPERTY = "name";
+  public static final String AUDIO_KEY_PROPERTY = "audioKey";
+  public static final String CREATOR_ID_PROPERTY = "creatorId";
+  public static final String PLACE_ID_PROPERTY = "placeId";
+  public static final String IS_PUBLIC_PROPERTY = "isPublic";
+  public static final String COORD_PROPERTY = "coord";
+  public static final String DESC_PROPERTY = "desc";
+  public static final String LENGTH_PROPERTY = "length";
+  public static final String IMG_KEY_PROPERTY = "imgKey";
+  
+  @Override
+  public void savePlaceGuide(String creatorId, PlaceGuide placeGuide) {
+    Entity placeGuideEntity = createPlaceGuideEntity(placeGuide);
+    datastore.put(placeGuideEntity);
+  }
```

Comment:
Why is `creatorId` a parameter for this function? It's not used, and placeGuide contains a creatorId field anyway.

Reply:
Removed creatorId from parameter.